cardano/src/address.rs

```diff
@@ -333,6 +456,10 @@ impl ExtendedAddr {
         ExtendedAddr::new(AddrType::ATPubKey, SpendingData::PubKeyASD(xpub), Attributes::new_bootstrap_era(None))
     }
 
+    pub fn to_address(&self) -> Addr {
+        Addr(cbor!(self).unwrap()) // unwrap should never fail from strongly typed extended addr to addr
```

Comment:
it is the cbor serialization that can fails. But if it fails here it means the user does not have anymore memory to _extend_ the `Vec` used in `cbor_event` to write the bytes of the cbor encoded address.